tools/accuracy_checker/data/test_models/pytorch_model/samplenet.py

```diff
@@ -14,13 +14,13 @@
 limitations under the License.
 """
 
-import torch.nn as nn
+from torch import nn
 import torch.nn.functional as F
 
 
 class SampLeNet(nn.Module):
     def __init__(self):
-        super(SampLeNet, self).__init__()
+        super().__init__()
```

Comment:
please return back, it is pytorch syntaxis
https://pytorch.org/docs/stable/generated/torch.nn.Module.html

Reply:
Done